.github/workflows/test.yml

```diff
@@ -18,14 +18,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Download wheels from commit OpenAssetIO feature branch
+      - name: Download wheels from OpenAssetIO PR
         uses: dawidd6/action-download-artifact@v2
         with:
           workflow: build-wheels.yml
           workflow_conclusion: success
           name: openassetio-wheels
           repo: OpenAssetIO/OpenAssetIO
           path: deps
+          # TODO(DF): Remove once upstream merged.
+          pr: 968
```

Comment:
Oh cool, i didn't know you could do this by pr number, lets you know what's going on more clearly.